homeassistant/components/homematicip_cloud.py

```diff
@@ -153,24 +191,19 @@ def available(self):
         """Device available."""
         return not self._device.unreach
 
-    def _generic_state_attributes(self):
+    @property
+    def device_state_attributes(self):
         """Return the state attributes of the generic device."""
         laststatus = ''
         if self._device.lastStatusUpdate is not None:
             laststatus = self._device.lastStatusUpdate.isoformat()
         return {
-            ATTR_HOME_LABEL: self._home.label,
+            ATTR_HOME_NAME: self._home.name,
             ATTR_DEVICE_LABEL: self._device.label,
```

Comment:
We re storing way too many things in the attributes here. Attributes should only be used to store things that describe the STATE of a device. I am not sure how all these other things came in here, but they should be removed.

Example of a good attribute: a light has the state "on". The attributes explain that the color is red.

Anything that does not explain the state, should be removed. Examples are unreachable, firmware state or status update (this one is already described by looking at the last_updated attribute of the state).

Reply:
I will remove all unnecessary things from the sensors as well as the status sensor for components. You are right it might confuse users.